### ISSUE TITLE
group-tab.html column width CSS no longer working

### ISSUE THREAD

**Issue by irvinm** [piroor/treestyletab#2683]
## Short description
On group-tab.html, when there are more than 1 column of tabs, the following CSS is no longer working.  It looks like it is defaulting back to 20em. (Discussion: https://github.com/piroor/treestyletab/issues/2607)

```
/*
   Change the column width once there is more than 1 column
   https://github.com/piroor/treestyletab/issues/2607
*/
:root.group-tab { --column-width: 40em; }
```

## Steps to reproduce

 1. Start Firefox with clean profile.
 2. Install TST.
 3. Create a group-tab.html.
 4. Create enough tabs to require more than 1 column.

## Expected result
CSS would format column usage of multiple columns.

## Actual result
CSS has no effect and the default 20em is being used.

## Environment

 * Platform (OS): Windows 2004 (x64)
 * Version of Firefox: FF 79.0
 * Version (or revision) of Tree Style Tab: TST 3.5.20

**Comment by irvinm** [comment]
@piroor I went back and verified that TST 3.5.19 worked fine.

**Comment by piroor** [comment]
Sorry, this is a regression from #2678 and the commit 24313b3 should fix this.

**Comment by irvinm** [comment]
Fixed and verified in 3.5.21.
